torch/_inductor/compile_fx.py

```diff
@@ -189,7 +189,7 @@ def _unlift_graph(mod, gm, graph_signature):
     outputs = list(gm.graph.nodes)[-1].args[0]
     mutated_outputs = []
     for out in outputs:
-        if out in graph_signature.buffers_to_mutate:
+        if str(out) in graph_signature.buffers_to_mutate:
```

Comment:
cc @angelayi - do you happen to know why inductor has its own unlift()` function? One thing I was wondering if is this code could reasonable be shared with `ExportedProgram.module()`.

I think this part was slightly broken, because `graph_signature.buffers_to_mutate` contains a bunch of strings (the name of each node), but `out` here is the fx node itself.